### ISSUE TITLE
sonic frontiers is unable to be detected even when using the latest version of hedgemodmanager

### ISSUE THREAD

**Issue by breezeepeezee** [thesupersonic16/HedgeModManager#657]
i've tried everything from reinstalling frontiers, to putting hedge mod manager in the same directory as frontiers, and i still can't seem to get frontiers detected no matter how hard i try.

[HMM_Snapshot_00072023171920.txt](https://github.com/thesupersonic16/HedgeModManager/files/11927141/HMM_Snapshot_00072023171920.txt)

HMM Info:
```
    Version: 7.10-0
    Args: 
    StartDir: E:\SteamLibrary\steamapps\common\SonicFrontiers\
    Process Level: User
    Game: No Game Detected!
    Update Status: Up To Date
```

Exception:
```
    Type: ArgumentNullException
    Message: Value cannot be null.
Parameter name: source
    Source: System.Core
    Function: TSource FirstOrDefault[TSource](System.Collections.Generic.IEnumerable`1[TSource], System.Func`2[TSource,System.Boolean])
    StackTrace: 
       at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
       at HedgeModManager.Epic.SearchForGamesHeroic() in E:\Git\hedgemodmanager\HedgeModManager\Epic.cs:line 89
       at HedgeModManager.Epic.SearchForGames() in E:\Git\hedgemodmanager\HedgeModManager\Epic.cs:line 43
       at HedgeModManager.GameInstall.SearchForGames(String preference) in E:\Git\hedgemodmanager\HedgeModManager\Game.cs:line 275
       at HedgeModManager.HedgeApp.Main(String[] args) in E:\Git\hedgemodmanager\HedgeModManager\HedgeApp.xaml.cs:line 216
    InnerException: 
```

**Comment by breezeepeezee** [comment]
ok so if you've had legendary launcher installed and want to completely uninstall it, don't just remove the exe itself.

what you **should** do is delete the .config folder on your user profile. that helped solve the issue for me <3

**Comment by thesupersonic16** [comment]
This issue was caused by a blank legendary configuration file and has now been fixed in the development channel.

Fixed 33f0dd594fe016bc68965f4bae7e88ba5d5971e8
